Updated incorrect powershell

ryanbor opened the pull request:
Updated incorrect powershell that has caused multiple cases as they need to use the New-AzureVnetGateway command instead Net-AzureVirtualNetworkGateway command as it will cause the connection to fail when created as it doesn't create as a primary gateway in the classic Vnet. Since this is for Classic only this affects this.

azurecla commented:
Hi __@CawaMS__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

ericspooner committed:
Updated incorrect powershell

Updated incorrect powershell that has caused multiple cases as they need to use the New-AzureVnetGateway command instead Net-AzureVirtualNetworkGateway command as it will cause the connection to fail when created as it doesn't create as a primary gateway in the classic Vnet. Since this is for Classic only this affects this.
